index.js

```diff
@@ -346,6 +359,16 @@ class GitlabScm extends Scm {
                 token
             })
             .then(response => {
+                if (response.statusCode === 404) {
+                    throwError(`Cannot find repository ${checkoutUrl}`, response.statusCode);
+                }
+                if (response.statusCode !== 200) {
+                    throwError(
+                        `STATUS CODE ${response.statusCode}: ${JSON.stringify(response.body)}`,
+                        response.statusCode
+                    );
+                }
```

Comment:
That might be a situation that should not be error on 2xx, so I remove this block.